src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -167,3 +167,33 @@ const applicantRelationshipEventMap = {
 export function recordApplicantRelationship({ formData: { onBehalfOf } }) {
   return <UserInteractionRecorder eventRecorder={recordEvent} selectedValue={onBehalfOf} trackingEventMap={applicantRelationshipEventMap}></UserInteractionRecorder>;
 }
+
+/**
+ * Provides an issue example tooltip with the provided text
+ * @param {string} Example text
+ */
+export const getIssueLabel = (labelText, exampleText) => {
+  return (<div><p>{labelText}</p><p><i>{exampleText}</i></p></div>);
+};
+
+export const transcriptReleaseLabel = getIssueLabel('Release of transcripts', 'The school won’t release your transcripts.');
+
```

Comment:
nit: I'd prefer to not have the extra line breaks between these. But I can live with this if others feel strongly.